### ISSUE TITLE
Error 105, Error 1 -- Automule issues

### ISSUE THREAD

**Issue by knarglegarth** [kolton/d2bot-with-kolbot#1936]
![](https://i.gyazo.com/e84a886c902fc3637ac63afb4a349606.png)

I'm lost as to what to do here. 

I noticed that in the screenie, I added an extra ]. The line actually reads like this:

// List of profiles that will mule items. Example: enabledProfiles: ["profile 1", "profile 2"],
			enabledProfiles: ["TW", TW2", "TW3"],

**Comment by Jestah66** [comment]
A shot in the dark here but is the capitalization of the profiles in d2bs iDentIcAl to what you put in automule.js?

Mine -  enabledProfiles: ["Astrid", "Lagertha", "GobSmacked", "GodSmacked"],

**Comment by knarglegarth** [comment]
Yes, I followed the punctuation. My assumption was that I was missing a **]** as it said in the error, but I didn't see where I would be missing one.

**Comment by 5noop** [comment]
Line 18 you are missing a "

**Comment by knarglegarth** [comment]
> Line 18 you are missing a " just before TW2

Awesome, so I got that part to work. But as far as configuring the mule profile... I only have 3 keys. How do I configure the mule profile to close one profile and begin itself?

**Comment by mf022** [comment]
https://github.com/blizzhackers/documentation/blob/master/kolbot/Automule.md#step-by-step-guide-to-muling-in-kolbot
check the **// Stop a profile prior to muling** section

**Comment by knarglegarth** [comment]
When I get home, I will definitely try that. I really appreciate the help, dudes.

**Comment by knarglegarth** [comment]
Just got home. I changed it to stop profile "TW3". When I pressed "5" to initiate the mule process with either TW or TW2, then TW3 closed as it was supposed to and switched to the MULE profile.
However, if I am on TW3, and I press "5" to initiate the automule, it just opens the MULE profile and of course clashes with the cd keys already open.

Do I need an additional fourth key just for the mule?

**Comment by Jestah66** [comment]
You should probably get a fourth key if you don't want to have to edit automule.js every time you want to mule the char that you specify for closure.  $17 bucks is worth it just for the convenience.  I run three bots but have 5 keys for different purposes.

**Comment by mf022** [comment]
>  I changed it to stop profile "TW3". ...
> However, if I am on TW3, and I press "5" to initiate the automule, it just opens the MULE profile and of course crashes with the cd keys already open.

Automule need 2 chars in the game.

Maybe you can set different mule profiles, smth like the settings for different realms with "Mule1" and "Mule2" (see the [guide](https://github.com/blizzhackers/documentation/blob/master/kolbot/Automule.md#multiple-mules-and-multiple-realm-muling)), so TW3 will be set in "Mule2", where you can stop other profile (TW1 or TW2).

Remember to set "true" the release of the key.

**Comment by knarglegarth** [comment]
Thanks a ton, everybody. Everything is working well now!
